packages/cel/biome.json

```diff
@@ -5,30 +5,6 @@
     "ignore": ["dist", "src/gen", "src/grammar.ts"]
   },
   "linter": {
-    "include": [
-      "src/type.ts",
-      "src/type.test.ts",
-      "src/std/math.ts",
-      "src/errors.ts",
-      "src/std/time.ts",
-      "src/std/cast.ts",
-      "src/std/logic.ts",
-      "src/ext/strings.ts",
-      "src/func.ts",
-      "src/eval.ts",
-      "src/value/adapter.ts",
-      "src/access.ts",
-      "src/testing.ts",
-      "src/equals.ts",
-      "src/equals.test.ts",
-      "src/field.ts",
-      "src/value/empty.ts",
-      "src/proto.ts",
-      "src/list.ts",
-      "src/list.test.ts",
-      "src/map.ts",
-      "src/map.test.ts",
-      "src/uint.ts"
-    ]
+    "ignore": ["src/parser.ts"]
```

Comment:
This is generated. As an aside, can we use `jsonc` for biome config. We can add comments and it'll be easier keep track of such things in code.

Reply:
There were some problems with using `jsonc`. Maybe we'll upgrade to v2 at some point and can revisit this.